dex/msgjson/types.go

```diff
@@ -32,33 +32,34 @@ const (
 	RPCArgumentsError                 // 16
 	RPCTradeError                     // 17
 	RPCCancelError                    // 18
-	SignatureError                    // 19
-	SerializationError                // 20
-	TransactionUndiscovered           // 21
-	ContractError                     // 22
-	SettlementSequenceError           // 23
-	ResultLengthError                 // 24
-	IDMismatchError                   // 25
-	RedemptionError                   // 26
-	IDTypeError                       // 27
-	AckCountError                     // 28
-	UnknownResponseID                 // 29
-	OrderParameterError               // 30
-	UnknownMarketError                // 31
-	ClockRangeError                   // 32
-	FundingError                      // 33
-	CoinAuthError                     // 34
-	UnknownMarket                     // 35
-	NotSubscribedError                // 36
-	UnauthorizedConnection            // 37
-	AuthenticationError               // 38
-	PubKeyParseError                  // 39
-	FeeError                          // 40
-	InvalidPreimage                   // 41
-	PreimageCommitmentMismatch        // 42
-	UnknownMessageType                // 43
-	AccountClosedError                // 44
-	MarketNotRunningError             // 45
+	RPCWithdrawError                  // 19
+	SignatureError                    // 20
+	SerializationError                // 21
+	TransactionUndiscovered           // 22
+	ContractError                     // 23
+	SettlementSequenceError           // 24
```

Comment:
At some point we need to look at which errors are for which subsystems (dcrdex comms vs dexcctl) and separate into separate enums.